scripts/Ionic.js

```diff
@@ -119,33 +127,35 @@ async function targetFACCheck(tokenAddress, amount) {
   );
 
   // Liquidity Check for targetFoundryTokken in TargetFundManager
-  const targetFoundryAssetLiquidity = await targetFoundryTokenContract.balanceOf(
-    targetFundMangerContract.address
-  );
+  const targetFoundryAssetLiquidity =
+    await targetFoundryTokenContract.balanceOf(
+      targetFundMangerContract.address
+    );
 
-  if (isTargetTokenFoundryAsset == true && Number(targetFoundryAssetLiquidity) > Number(amount)) {
-      return true;
-  }
-  else{
+  if (
+    isTargetTokenFoundryAsset == true &&
+    Number(targetFoundryAssetLiquidity) > Number(amount)
+  ) {
+    return true;
+  } else {
     return false;
   }
 }
 
-
 //check source toke is foundry asset
 async function isSourceRefineryAsset(tokenAddress, amount) {
   try {
     const isTokenFoundryAsset = await sourceFACCheck(tokenAddress);
 
-      let path = [tokenAddress, sourceFoundryTokenAddress];
-      const amounts = await sourceDexContract.getAmountsOut(amount, path);
-      const amountsOut = amounts[1];
+    let path = [tokenAddress, sourceFoundryTokenAddress];
+    const amounts = await sourceDexContract.getAmountsOut(amount, path);
+    const amountsOut = amounts[1];
 
-      if (isTokenFoundryAsset == false && Number(amountsOut) > 0) {
-        return true;
-      } else {
-        return false;
-      }
+    if (isTokenFoundryAsset == false && Number(amountsOut) > 0) {
+      return true;
+    } else {

```

Comment:
I think else { could be removed and return false directly for code optimization.